src/lib/Components/FilteredInfiniteScrollGrid/FilteredInfiniteScrollGrid.tsx

```diff
@@ -8,6 +9,8 @@ import { FiltersContainer as Filters } from "./Filters"
 interface Props {
   onRefetch: (params: object) => any
   filteredArtworks: any
+  fairSlug?: string
```

Comment:
hmm.. this is meant to be a generic component so adding fair specific props isn't great.
Maybe name this `entityData: {id: string, slug: string}`

Reply:
That being said I think it's ok for now

Reply:
Updated this to make it more generic